src/pages/docs/deployments/bulk-deployment-creation.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-09-13
+modDate: 2023-09-13
+title: Bulk deployment creation
+description: Octopus Deploy can use bulk deployment creation when deploying to multiple environments or tenants
+navOrder: 160
+---
+
+:::div{.hint}
+Bulk deployment creation will be available from Octopus Deploy **2023.4**
+:::
+
+When deploying a release to multiple environments or tenants through the Octopus Portal, the bulk deployment creation feature will immediately take you to a new server task which will create a deployment for each environment or tenant you are deploying to.
```

Comment:
Consider re-wording to avoid passive voice:

You will be redirected to the server task page upon deploying a release to multiple environments or tenants. There you will find each requested deployment   recorded as an indvidual task.

Reply:
A few thoughts: 

* Worth calling out that this is an Octopus Portal feature. 
* I was reluctant to introduce the term "requested" as a concept but given that we use it in code, it does make comms easier all round so I'm happy to use it.
* Worth disambiguating "deployment recorded as an indvidual task". I think server task when I see the word "task" and the user really just sees a summary

What about:

```suggestion
With bulk deployment creation, you will be redirected to a new server task upon deploying a release to multiple environments or tenants through Octopus Portal. There you will find each requested deployment as task summary items, which will display progress and any errors.
```